.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+* Check the [guidelines](#contribution-guidelines).
+* [Fork and clone](https://help.github.com/articles/fork-a-repo/) the [DevDocs repository](GitHub.com/magento/devdocs). [Sync](https://help.github.com/articles/syncing-a-fork/) as needed.
+* Write content and commit.  Need help, see our [templates](#templates).
+* Create a [pull request](https://help.github.com/articles/creating-a-pull-request/). Fill out as much info as possible and link any GitHub issues.
+
+**Note:** If you have not signed the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca), the PR will provide a link to complete. All contributors are required to submit the form and agree to the terms.
+
+**TIP:** If you are not sure where to start contributing, you can check GitHub issues: [help wanted](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [good first issue](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+
+## Contribution guidelines
+We use [Markdown](http://daringfireball.net/projects/markdown/) to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html).
+
+You can update, or add content to, existing topics in their respective versioned directories (for 2.0, 2.1, 2.2, and onward).
+
+The following guidelines may answer most of your questions and help you get started:
+
+1.  Check [existing pull requests](https://GitHub.com/magento/devdocs/pulls) and make sure you are not duplicating work!
+
+1.  For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
+
+1.  Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+
+    -   The DevDocs team can find the best home for your new topics and add it to the navigation.
+    -   If a topic has a symlink, you can remove it with git commands and add a new file. Copy and paste a previous version of the topic to get started.
+
+1.  Focus on the content and on creating useful information for your fellow Magento developers and community members. Don't forget to review your work for typos, formatting errors, or sentences that need clarifying before opening a pull request.
+
+1.  Use the following guidelines to help you with the writing process:
+
+    -   Define the goal of your topic. What exactly do you want to teach the reader?
+    -   Make the title of your topic reflect the content.
+    -   Keep your sentences concise and try to separate conceptual information from procedural steps.
+    -   Remember to use active voice (not passive), write in the present tense, and use a friendly tone in second person. For example, _"You can now view the output..."_.
```

Comment:
It seems as the example text would actually be "View the output..." so maybe not a great example? Judgment call.